src/metadata_validator.py

```diff
@@ -482,7 +482,8 @@ def validate_prop_value(self, prop_name, value, prop_def, msg_prefix):
             
             elif type == "array" or type == "value-list":
                 val = str(value)
-                arr = val.split("*") if "*" in val else val.split(",") if "," in val else [value]
+                list_delimiter = self.model.get_list_delimiter()
+                arr = val.split(list_delimiter) if list_delimiter in val else [value]
```

Comment:
Do we need to strip the leading and trailing spaces here? In case we got "TXT | PDF | JPEG"

Reply:
Yes, each item is stripped as shown below:
for item in arr:
                    val = item.strip() if item and isinstance(item, str) else item
                    result, error = check_permissive(val, permissive_vals, msg_prefix, prop_name)
                    if not result:
                        errors.append(error)